docs/io.cozy.files_metadata.md

```diff
@@ -0,0 +1,392 @@
+
+# Metadata on `io.cozy.files` documents
```

Comment:
[nit] Can we add a link to this document in the [README](https://github.com/cozy/cozy-doctypes/blob/master/docs/README.md)?

Reply:
Corrected in last commit